### DISCUSSION TITLE
ESP NSPanel internet access

### DISCUSSION THREAD

**Discussion by ucakca** [Blackymas/NSPanel_HA_Blueprint#1773] [General]
Hi! in the configuration of NSPanel after activating the internet connection the button jumps back to off.
Aside from that the Screen saver - Display time does not work.
Is there any solution experience?

**Comment by edwardtfn** [comment]
What do you mean by "activating the Internet connection"? Could you please provide a bit more of details? Perhaps some picture of the screen, or a step by step to reproduce this...

For the display time, please make sure your dim sleep value is not too close to 0. You can find this on the device page under Settings > Devices and Services > ESPHome.

**Reply by ucakca** [reply]
https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/153840514/c895b745-a64b-4191-8a89-f2059190f30e

![Screenshot_20240210-203958_Home Assistant.jpg](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/153840514/989b88b1-deb3-41d2-ac93-5eb70761a365)

Hi! Here I can see the issue with Internet Access and the brightness settings for displaying the time as screen saver. 

The settings for the brightness are taken over but without displaying the time.

**Reply by edwardtfn** [reply]
This "**Internet access**" entity is not provided by this project. This is probably provided by your router's integration.

About the display time when sleep, could you please [report as a bug in the issues area](https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/new/choose) and provide more details like the version of TFT, ESPHome and Blueprint shown in the boot screen?
